components/script/dom/performanceresourcetiming.rs

```diff
@@ -250,4 +249,9 @@ impl PerformanceResourceTimingMethods for PerformanceResourceTiming {
     fn ResponseEnd(&self) -> DOMHighResTimeStamp {
         Finite::wrap(self.response_end)
     }
+
+    // https://w3c.github.io/resource-timing/#dfn-duration
```

Comment:
We will need to revert this change and ensure that we pass appropriate values to PerformanceEntry::new_inherited instead, so that PerformanceEntry::Duration and PerformanceEntry::StartTime return the expected values..